### PR TITLE
Created new configs for linkace, storm and YourSpotify

### PR DESCRIPTION
**Pull request by Jurrer**
[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

------------------------------

 - [x] I have read the [contributing](https://github.com/linuxserver/reverse-proxy-confs/blob/master/.github/CONTRIBUTING.md) guideline and understand that I have made the correct modifications

------------------------------

## Description
I am sharing my configs that I found useful.

## Benefits of this PR and context

It adds new functionality. I hope it will save people some time.

## How Has This Been Tested?

Successfully tested on own lsio nginx instance. Did not test authentication options.

## Source / References

[YourSpotify](https://github.com/Yooooomi/your_spotify)
[LinkAce](https://github.com/Kovah/LinkAce)
[Storm](https://github.com/relvacode/storm)

### CODE REVIEW

File: linkace.subdomain.conf.sample [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    # enable to indicate to search engines to not index this site
+    # add_header X-Robots-Tag "noindex, nofollow, nosnippet, noarchive";
+    
+    # Content security headers for Laravel
+    add_header X-Frame-Options "SAMEORIGIN";
+    add_header X-XSS-Protection "1; mode=block";
+    add_header X-Content-Type-Options "nosniff";
+
+    proxy_set_header X-Forwarded-For $remote_addr;
+    proxy_set_header X-Forwarded-Proto $scheme;
```
Comment (drizuid) [inline linkace.subdomain.conf.sample:21]: remove

File: linkace.subdomain.conf.sample [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    # add_header X-Robots-Tag "noindex, nofollow, nosnippet, noarchive";
+    
+    # Content security headers for Laravel
+    add_header X-Frame-Options "SAMEORIGIN";
+    add_header X-XSS-Protection "1; mode=block";
+    add_header X-Content-Type-Options "nosniff";
+
+    proxy_set_header X-Forwarded-For $remote_addr;
+    proxy_set_header X-Forwarded-Proto $scheme; 
+    proxy_set_header Host $host;
```
Comment (drizuid) [inline linkace.subdomain.conf.sample:22]: remove, duplicate from proxy.conf

File: linkace.subdomain.conf.sample [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+    # enable to indicate to search engines to not index this site
+    # add_header X-Robots-Tag "noindex, nofollow, nosnippet, noarchive";
+    
+    # Content security headers for Laravel
+    add_header X-Frame-Options "SAMEORIGIN";
+    add_header X-XSS-Protection "1; mode=block";
+    add_header X-Content-Type-Options "nosniff";
+
+    proxy_set_header X-Forwarded-For $remote_addr;
```
Comment (drizuid) [inline linkace.subdomain.conf.sample:20]: confirm this is needed, remove it and test, overrides `proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;` which is probably what is needed

File: linkace.subdomain.conf.sample [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+    server_name linkace.*;
+
+    # enable to indicate to search engines to not index this site
+    # add_header X-Robots-Tag "noindex, nofollow, nosnippet, noarchive";
+    
+    # Content security headers for Laravel
+    add_header X-Frame-Options "SAMEORIGIN";
+    add_header X-XSS-Protection "1; mode=block";
+    add_header X-Content-Type-Options "nosniff";
```
Comment (drizuid) [inline linkace.subdomain.conf.sample:18]: we do not enforce security policies on users in this manner, they can uncomment these things in ssl.conf

File: linkace.subdomain.conf.sample (hunk 3 above)
Comment (Jurrer) [reply linkace.subdomain.conf.sample]: not needed, removed it

File: linkace.subdomain.conf.sample [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+server {
+    listen 443 ssl http2;
+    listen [::]:443 ssl http2;
+
+    server_name linkace.*;
+
+    # enable to indicate to search engines to not index this site
+    # add_header X-Robots-Tag "noindex, nofollow, nosnippet, noarchive";
+
```
Comment (nemchik) [inline linkace.subdomain.conf.sample:14]: I would prefer not including these here for the sake of keeping the confs uniform. It is reasonable to add headers to individual proxies, but it's more likely that this header would be added to the `ssl.conf`.

File: storm.subdomain.conf.sample [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+server {
+    listen 443 ssl http2;
+    listen [::]:443 ssl http2;
+
+    server_name storm.*;
+
+    # enable to indicate to search engines to not index this site
+    # add_header X-Robots-Tag "noindex, nofollow, nosnippet, noarchive";
+
```
Comment (nemchik) [inline storm.subdomain.conf.sample:15]: Same comment from linkace regarding this header.

File: your-spotify.subdomain.conf.sample [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+
+server {
+    listen 443 ssl http2;
+    listen [::]:443 ssl http2;
+
+    server_name  your-spotify.*;
+
+    # enable to indicate to search engines to not index this site
+    #add_header X-Robots-Tag "noindex, nofollow, nosnippet, noarchive";
+
```
Comment (nemchik) [inline your-spotify.subdomain.conf.sample:17]: Same comment from linkace regarding this header.

File: your-spotify.subdomain.conf.sample [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+        set $upstream_port 3000;
+        set $upstream_proto http;
+        proxy_pass $upstream_proto://$upstream_app:$upstream_port;
+
+    }
+
+}
+
+
+server {
```
Comment (nemchik) [inline your-spotify.subdomain.conf.sample:57]: This should be a separate file, and comments at the top of both indicating if they are both required that you should enable them both.

File: your-spotify.subdomain.conf.sample [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+    listen 443 ssl http2;
+    listen [::]:443 ssl http2;
+
+    server_name  your-spotify-server.*;
+
+    # enable to indicate to search engines to not index this site
+    #add_header X-Robots-Tag "noindex, nofollow, nosnippet, noarchive";
+
```
Comment (nemchik) [inline your-spotify.subdomain.conf.sample:65]: Same comment from linkace regarding this header.

**Review by nemchik — CHANGES_REQUESTED** [review]

File: your-spotify.subdomain.conf.sample (hunk 8 above)
Comment (Jurrer) [reply your-spotify.subdomain.conf.sample]: done

**Review by nemchik — APPROVED** [review]
lgtm, thanks!

**Pull request merged**